src/i18n/nl.json

```diff
@@ -1242,5 +1242,25 @@
         "link": "Je videolink voldoet niet aan de vereisten."
       }
     }
+  },
```

Comment:
Don't forget to add the FR and DE translations.
You can copy them from the udb3-angular repo :)
https://github.com/cultuurnet/udb3-angular/blob/main/src/core/translations/french-translations.constant.js

https://github.com/cultuurnet/udb3-angular/blob/main/src/core/translations/german-translations.constant.js